admin/release.py

```diff
@@ -697,11 +757,22 @@ def publish_artifacts_main(args, base_path, top_level):
                     scratch_directory=scratch_directory.child('pip'),
                     target_bucket=options['target'],
                 ),
-                copy_tutorial_vagrant_box(
-                    target_bucket=options['target'],
-                    dev_bucket='clusterhq-dev-archive',
+                Effect(
+                    CopyS3Keys(
+                        source_bucket=DEV_ARCHIVE_BUCKET,
+                        source_prefix=vagrant_prefix,
+                        destination_bucket=options['target'],
+                        destination_prefix=vagrant_prefix,
+                        keys=[box_name],
+                    )
+                ),
+                publish_vagrant_metadata(
                     version=options['flocker-version'],
-                )
+                    box_url=box_url,
+                    scratch_directory=scratch_directory.child('vagrant'),
+                    box_name=box_type,
+                    target_bucket=options['target'],
+                ),
```

Comment:
Okay, reading this I see that probably the same code is used to publish both the dev and tutorial boxes.  But there's a separate tool for publishing dev boxes whereas the tutorial boxes are published along with all the other stuff in the main release tool.